spec/rubocop/cop/style/redundant_parentheses_spec.rb

```diff
@@ -52,6 +52,8 @@
   it_behaves_like 'redundant', '(true)', 'true', 'a literal'
   it_behaves_like 'redundant', '(false)', 'false', 'a literal'
   it_behaves_like 'redundant', '(/regexp/)', '/regexp/', 'a literal'
+  it_behaves_like 'plausible', '("x"; "y")'
```

Comment:
I mean - don't you need an assignment operator preceding those expressions to be sure the parens are needed?